base_user_role/README.rst

```diff
@@ -116,6 +120,7 @@ Contributors
 * Kevin Khao <kevin.khao@akretion.com>
 * Tatiana Deribina <tatiana.deribina@sprintit.fi> (https://sprintit.fi)
 * Guillem Casassas <guillem.casassas@forgeflow.com>
+* Guillaume Pothier <gpothier@caligrafix.cl>
```

Comment:
You should not update this file (README.rst) as it is an autogenerated file, updated when the PR is merged.

you can change fragments present in the readme folder.